tao_compiler/mlir/xla/ral/context/common_context_impl_mkldnn.cc

```diff
@@ -407,9 +415,42 @@ void ral_conv(ExecutionContext* ctx, opaque_t /*stream_handle*/,
     // reorder to dst format
     y.reorder_to(params.dst);
   } else {
-    ideep::convolution_forward::compute<true>(
-        params.src, params.weight, params.dst_dims, params.dst, params.strides,
-        params.dilates, params.padding_l, params.padding_r, params.groups);
+    if (params.weight_is_const) {
+      ideep::convolution_forward_params conv_params;
+      ideep::convolution_forward::prepare<true>(
+          conv_params, params.src, params.weight, params.dst_dims, params.dst,
+          params.strides, params.dilates, params.padding_l, params.padding_r,
+          params.groups);
+
+      ideep::tensor packed_weight;
+      std::string unique_name = "tao_ral.cpu.mkldnn_conv_" +
+                                tao::ral::TaoTypeNameHelper<Tinput>::Invoke();
+      auto state = ctx->getOrCreateResource<MkldnnConvState>(
+          unique_name, []() { return new MkldnnConvState; });
```

Comment:
Why need to create a MkldnnConvState for each Tinput?

Reply:
Just to reduce the change of lock conflict.